pages/travel.md

```diff
@@ -5,6 +5,10 @@ description: Information for participants traveling to Princeton
 permalink: /travel/
 ---
 
+## COVID-19 Policy
+All participants must to adhere to current Princeton University [guidelines, policies, and protocols for visitors and conferences](https://covid.princeton.edu/policies).
```

Comment:
When you say "current" you mean current at the time of the workshop right?

Reply:
Yes. I changed the wording slightly. Let me know if it's clearer

Reply:
Looks good.